mrp_project/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+MRP Project Link
+================
+
+It links projects and tasks to *manufacturing orders* (MO) and
+*work orders* (WO).
+
+Usage
+=====
+
+In a manufacturing order (MO), you can select a project to be attached to it.
+If none is selected, when the MO is confirmed, a project is automatically
```

Comment:
If none is selected, a project is automatically created when the MO is confirmed.